frontend/src/test/scala/bloop/engine/FileWatchingSpec.scala

```diff
@@ -0,0 +1,148 @@
+package bloop.engine
+
+import java.io.{ByteArrayOutputStream, PrintStream}
+import java.nio.file.Files
+
+import bloop.Project
+import bloop.cli.{CliOptions, Commands}
+import bloop.tasks.{CompilationHelpers, ProjectHelpers}
+import bloop.tasks.ProjectHelpers.{RootProject, noPreviousResult, withState}
+import org.junit.Test
+
+class FileWatchingSpec {
+  @scala.annotation.tailrec
+  final def readCompilingLines(target: Int, msg: String, out: ByteArrayOutputStream): Int = {
```

Comment:
I'd rename `readCompilingLines` to `outputContains`, and `target` to `times` or `expected`. WDYT?

Reply:
Sounds goooooooood.